src/commands/Tools/topinvites.ts

```diff
@@ -0,0 +1,34 @@
+import { CommandStore, KlasaMessage } from 'klasa';
+import { SkyraCommand } from '../../lib/structures/SkyraCommand';
+import { Invite } from 'discord.js';
+
+export default class extends SkyraCommand {
+
+	public constructor(store: CommandStore, file: string[], directory: string) {
+		super(store, file, directory, {
+			cooldown: 10,
+			description: language => language.tget('COMMAND_TOPINVITES_DESCRIPTION'),
+			extendedHelp: language => language.tget('COMMAND_TOPINVITES_EXTENDED'),
+			requiredGuildPermissions: ['MANAGE_GUILD']
+		});
+	}
+
+	public async run(message: KlasaMessage) {
+		const invites = await message.guild!.fetchInvites();
+		const topTen = invites
+			.filter(inv => this.filterInvites(inv))
```

Comment:
`.filter(this.filterInvites)`

Reply:
Actually, this would require binding of the method due to `this` becoming undefined with an unbound method

Reply:
so bind it? 🤣

Reply:
Exactly what Pyro said, that's what I wrote at first. @kyranet's call on what he prefers but considering he upvoted Pyro's comment I assume he prefers the current way.

Reply:
I prefer bind, but I'm fine with this one as well.